test/unit/annotation_layer_spec.js

```diff
@@ -172,4 +188,35 @@ describe('Annotation layer', function() {
       expect(borderStyle.verticalCornerRadius).toEqual(0);
     });
   });
+
+  describe('FileAttachmentAnnotation', function() {
+    var loadingTask;
+    var annotations;
+
+    beforeEach(function() {
+      var pdfUrl = combineUrl(window.location.href,
+                              '../pdfs/annotation-fileattachment.pdf');
+      loadingTask = PDFJS.getDocument(pdfUrl);
+      waitsForPromiseResolved(loadingTask.promise, function(pdfDocument) {
```

Comment:
I tried this, but unfortunately it does not seem to work. If I don't explicitly use `waitsForPromiseResolved` everywhere, the `annotations` variable is undefined when we reach the unit test.